### PR TITLE
Change on twitter widget

### PR DESCRIPTION
**Pull request by teehamaral**
Pull request related to issue #190

### CODE REVIEW

File: templates/public/index.haml [hunk 1]
```diff
@@ -265,7 +265,7 @@
               .brick-align{style:"padding: 50px 10px;"}
                 %a.twitter-follow-button{ href:'https://twitter.com/{{ org|config:"twitter_handle" }}', data-show-count:"true", data-lang:"en"}
 
-                %a.twitter-timeline{ href:'https://twitter.com/{{ org|config:"twitter_handle" }}', data-widget-id:'{{ org|config:"twitter_user_widget" }}', width:"340px", height:"600px", data-chrome:"noheader nofooter", data-link-color:"#4aaad4" }
```
Comment (norkans7) [inline templates/public/index.haml]: we can keep the data-chrome attributes as well here

**Comment by norkans7** [comment]
@teehamaral can you update the widget in the join template as well?

**Comment by teehamaral** [comment]
Yes @norkans7, I also keep the chrome-data attribute.

**Comment by norkans7** [comment]
Looks good. thanks @teehamaral

**Comment by teehamaral** [comment]
Thanks @norkans7.

**Pull request merged**